document-models/document-drive/schema.graphql

```diff
@@ -1,91 +1,68 @@
 type FolderNode {
-    path: String!
+    id: String!
```

Comment:
just asking for my own understanding, why are we using String! instead of ID! for the type of the id in these file and folder nodes?